tests/Base.nut

```diff
@@ -0,0 +1,81 @@
+// MIT License
```

Comment:
Not sure I understand why it prevents us from turning it into a base class. Can you elaborate please?